cogs/auctions.py

```diff
@@ -577,6 +579,13 @@ async def info(self, ctx, auction: AuctionConverter):
                 f"**Bid Increment:** {auction['auction_data']['bid_increment']:,} Pokécoins",
             )
 
+        embed.add_field(
+            name="Current Moves",
+            value="No Moves"
+            if len(pokemon.moves) == 0
+            else "\n".join(self.bot.data.move_by_number(x).name for x in pokemon.moves),
+        )
```

Comment:
Since this is an embed that is identical in other files (market & trading) it may be a good idea to make this embed in a seperate file that can be accessed. This way your code is more DRY (Don't Repeat Yourself).

Reply:
Good point, but I'm not really sure what a good way to do this would be since all those embeds have different designs. Which is why I think this is fine, for now at least

Reply:
maybe something like 

embed = moves.addCurrentMoves(embed)

just an idea?

Reply:
Oo, that works. I can add it to the utils file. Thank you

Reply:
Done! Does that look ok?

Reply:
Looks awesome! I'll resolve this conversation :)